src/main/java/com/adyen/model/nexo/SaleData.java

```diff
@@ -75,7 +79,8 @@
      * The Sale to acquirer data.
      */
     @XmlElement(name = "SaleToAcquirerData")
-    protected String saleToAcquirerData;
+    @JsonAdapter(SaleToAcquirerDataModelAdapterSerializer.class)
+    protected SaleToAcquirerDataModel saleToAcquirerData;
```

Comment:
We can skip Model in name of class and keep it  SaleToAcquirerData because package name already has "model"

Reply:
good point I agreed